### ISSUE TITLE
[Feature request] Swap history graph

### ISSUE THREAD

**Issue by w8jcik** [AstraExt/astra-monitor#160]
Depending on how a PC is used, sometimes swap usage is as important as normal memory usage.

It is not only about systems with limited RAM, but also powerful machines that are dealing with a heavy load.

Currently there is no option to display a bar/graph for swap usage on the status bar.

**Comment by ljuzig** [comment]
yeah, I guess that for someone it might be crucial to keep track of the swap, I try to add it by the next release.
